mdsuite/database/simulation_database.py

```diff
@@ -398,14 +402,19 @@ def resize_datasets(self, structure: dict):
 
                 # get the correct maximum shape for the dataset -- changes if an
                 # experiment property or an atomic property
-                if len(dataset_information[:-1]) == 1:
-                    axis = 0
-                    expansion = dataset_information[0] + db[identifier].shape[0]
-                else:
-                    axis = 1
-                    expansion = dataset_information[1] + db[identifier].shape[1]
+                try:
+                    if len(dataset_information[:-1]) == 1:
+                        axis = 0
+                        expansion = dataset_information[axis] + db[identifier].shape[axis]
```

Comment:
Can you move this outside the `if .. else` ?

Reply:
resolve [move statement out of if else](https://github.com/zincware/MDSuite/pull/587/commits/bfbf4146cc8ea2477f68a8f800f911e0bf1540f2)